services/m3dbnode/config/config_test.go

```diff
@@ -186,6 +186,16 @@ pooling:
         capacity: 4096
         lowWatermark: 0.01
         highWatermark: 0.02
+    tagArrayPool:
+        size: 4096
```

Comment:
Might want to use a little higher considering this is kind of like an example config.  Perhaps 65536?

Reply:
Sure thing

Reply:
done